code/tools/make_munki_mpkg.sh

```diff
@@ -897,6 +957,10 @@ for pkg in $ALLPKGS ; do
         "config")
             ver="1.0"
             ;;
+        "clientcert")
+            ver="1.0"
+            SCRIPTS="${MUNKIROOT}/code/tools/pkgresources/Scripts_nothing"
```

Comment:
Why are you including a postinstall script that does nothing? Why not have no script?

Reply:
IIRC, if the SCRIPTS variable wasn't specified, the build script just included whatever script was last set for this variable, so at the time I worked around the problem by stubbing out this do nothing script, similar to Scripts_python, under the assumption that a script had to be defined.

But in looking at this again, I see that was a bad assumption, and it's better to just set the SCRIPTS variable to null. 

I'll make this change and test again.

Reply:
I find that this entire script is getting too big and unwieldy, and probably desperately needs a re-think.

Reply:
Okay, that change, `SCRIPTS=""`, worked fine (checked resulting package in Suspicious Package).

Note that your `"bootstrap") `and `"config")` cases don't define SCRIPTS either (which is template I used first time in my code, before stubbing out a script), so they're just inheriting whatever this variable is set to, in this case the Scripts_python do nothing postinstall script is included in these component packages. Do you want to me also define `SCRIPTS=""` for these as well?

Reply:
Yes, please.

Reply:
Changes pushed.